lib/tasks/locale.rake

```diff
@@ -274,7 +274,10 @@ namespace :locale do
       require Rails.root.join("lib/vmdb/gettext/domains")
 
       po_files = {}
-      Vmdb::Gettext::Domains.paths.each do |path|
+
+      # TODO: Domains for po and mo are added with the same path in Vmdb::FastGettextHelper.register_locales
+      # We need to uniq them for now, otherwise we find and merge the same po files.
+      Vmdb::Gettext::Domains.paths.uniq.each do |path|
```

Comment:
This avoids doing `rmsgcat -o ... locale/en/manageiq.po locale/en/manageiq.po` down below.  We should probably have a `po_paths` method that returns a unique set of paths since the current `paths` is returning paths for mo and po files.